Assets/NatShare/Plugins/Managed/NatShareNull.cs

```diff
@@ -10,17 +10,17 @@ namespace NatShareU.Platforms {
 
     public class NatShareNull : INatShare {
 
-        bool INatShare.Share (string text) {
+        bool INatShare.ShareText (string text, ShareCallback callback) {
```

Comment:
The `callback` function should be called in some way. Either from withing the `ShareXXX` function (but it will be called **before** the function has returned), or using a "temporary" script running a coroutine so as to call the `callback` at the end of the frame for instance.

Reply:
Good point. I've done that. If there aren't any other changes, please approve this PR.